aiodocker/swarm.py

```diff
@@ -1,20 +1,22 @@
-import json
-from typing import Optional, Dict
+from typing import Dict, List
+
+from .utils import clean_map
 
 
 class DockerSwarm(object):
     def __init__(self, docker):
         self.docker = docker
 
-    async def init(self,
-                   *,
-                   advertise_addr: Optional[str]=None,
-                   listen_addr: str="0.0.0.0:2377",
-                   force_new_cluster: bool=False,
-                   swarm_spec: Optional[Dict]=None
-                   ) -> str:
+    async def init(
+        self,
+        *,
+        advertise_addr: str=None,
+        listen_addr: str="0.0.0.0:2377",
+        force_new_cluster: bool=False,
+        swarm_spec: Dict=None
```

Comment:
I'd prefer more generic `typing.Mapping` or `typing.MutableMapping` here.
Q: Have we agreed with remove `Optional` for nullable parameters? (Maybe I have missed the conversation due to inactivity...)

Reply:
```Dict=None is like Optional[Dict]```, I'll change the type with mapping 👍

Reply:
Ah, so `=None` implies `Optional` in this case. Got it. :+1: